view/rooms/app.js

```diff
@@ -577,11 +577,14 @@ function createIframeEvent(channel, payload, bots, botId) {
     return 'Bot not found';
   }
 
-  iframedoc.getElementById(target)
-    .dispatchEvent(new CustomEvent(channel, {
-      detail: dispatchObj,
-    })
-  );
+  if (iframedoc.getElementById(target)) {
+    iframedoc.getElementById(target)
```

Comment:
can you call `getElementById` only once? doing stuff with DOM is rather expensive

Reply:
done @kamilsmuga